clang/test/SemaSYCL/Inputs/sycl.hpp

```diff
@@ -85,37 +85,37 @@ struct _ImplT {
   id<dim> Offset;
 };
 
-template <typename dataT, access::target accessTarget>
+using propertyListT = ext::oneapi::accessor_property_list<>;
+template <typename dataT, access::target accessTarget, typename propertyListT>
 struct DeviceValueType;
 
-template <typename dataT>
-struct DeviceValueType<dataT, access::target::global_buffer> {
+template <typename dataT, typename propertyListT>
```

Comment:
Why these propertyList changes? Seems unrelated to your patch

Reply:
These have been changes because of the change at line #118.

Reply:
Can you please explain why the change on L118 is required?

Reply:
@elizabethandrews please refer to the comments above about the motivation for this change. https://github.com/intel/llvm/pull/3892#discussion_r658166621

Reply:
Thanks for pointing that out. I'm not too familiar with `property_lists` and so I don't know if I am missing something obvious. From my reading, that discussion is about an alternate way to handle `property_lists`, suggested by @bader, to avoid compiler from having to infer and add attributes. Please correct me if I am mistaken @bader @zahiraam. 

Your patch however does not change `property_list` handling as far as I can tell. I understand we may do this change in the future these test header changes should be done whenever we implement that change i.e. we should change the test headers only when we change the actual headers in my opinion.